### PR TITLE
fix(i18n): minor tweaks to ja.json

### PR DESCRIPTION
**Pull request by watilde**
Hellooo👋 

Happy to see having `i18n`! I did a few minor tweaks to `ja.json`. They all are quite minor changes so feel free to arrange them if you like :)

Thanks (´ー｀)

### CODE REVIEW

File: locales/ja.json [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+  "Location of the npm cache.": "npmのキャッシュへのパス。",
   "Skip installation if a package is missing.": "パッケージがない場合は、インストールをスキップします。",
   "Path to user npmrc.": "ユーザーのnpmrcへのパス。",
-  "Execute string as if inside `npm run-script`.": "「npm run-script」の内部にあるかのように文字列を実行する。",
+  "Execute string as if inside `npm run-script`.": "「npm run-script」の内部にあるかのように文字列を実行します。",
   "Shell to execute the command with, if any.": "コマンドを実行するシェル（存在する場合）。",
-  "Generate shell code to use npx as the \"command not found\" fallback.": "「コマンドが見つかりません」フォールバックとして使用するコードを生成します。",
-  "Ignores existing binaries in $PATH, or in the local project. This forces npx to do a temporary install and use the latest version.": "$PATHまたはローカルプロジェクトの既存のバイナリを無視します。 これにより、npxは一時的なインストールを行い、最新バージョンを使用します。",
-  "npm binary to use for internal operations.": "npmバイナリを内部操作に使用します。",
+  "Generate shell code to use npx as the \"command not found\" fallback.": "コマンドが見つかりません。フォールバックとして使用するコードを生成します。",
```
Comment (zkat) [inline locales/ja.json]: This change reads weird to me. `"command not found"` is the term for the fallback itself. This sentence is the description for `--shell-auto-fallback`, and what it does is output shell code that will _then_ take care of things should you have `command not found`. There is no error when this happens. Am I just misreading what it means to put that in its own sentence?...

**Review by zkat — COMMENTED** [review]
I'm confused about one bit of the translation. Otherwise LGTM!

本当にありがとうございます！まだ日本語をよく話せません。（＾＿＾）；

File: locales/ja.json (hunk 1 above)
Comment (watilde) [reply locales/ja.json]: That's a good point! `「」` is basically used in a talk between humans([wikipedia in ja](https://ja.wikipedia.org/wiki/%E6%8B%AC%E5%BC%A7#.E9.89.A4.E6.8B.AC.E5.BC.A7.E3.80.8C.E3.80.8D)). We might use double quotes as is, here is an example:
+ https://github.com/vim-jp/lang-ja/blob/master/src/po/ja.po

I will add double quotes if it's fine :)

Comment (zkat) [reply locales/ja.json]: That sounds good!

Comment (watilde) [reply locales/ja.json]: Done(´ー｀)

**Pull request merged**